zombienet/0003-block-building-warp-sync/test-block-building-warp-sync.zndsl

```diff
@@ -0,0 +1,35 @@
+Description: Warp sync
```

Comment:
I'm a little bit confused why we have this extra test. Could we not at least merge them with the other one?

Reply:
We could, but I’d prefer to keep them separate, they are slightly different. I think having smaller tests is simpler in terms of debugging.

Please refer to: https://github.com/paritytech/substrate/pull/13078#issuecomment-1373506149 for some details.